### PR TITLE
feat(os): Added to steps to contribute and steps to run locally in readme

### PR DESCRIPTION
**Pull request by angelaliu22**
**Why**
When contributing to the OS I realized I didn't know how to run the repo locally. Team assisted via a Slack message. This is a good opportunity to add the process to the readme for future Flexpals to reference as well.

**What**
1. Added steps to contribute to the readme
2. Added steps to run the os locally

### CODE REVIEW

File: docs/process/development.md [hunk 1]
```diff
@@ -225,6 +225,16 @@ Details on pairing here.
...
+<<<<<<< Updated upstream
+=======
+#### Design review & scoping
+* **Duration**: As long as it takes (typically starting with 90 minutes)
+
+**Purpose**: To denmark the end of large scope design work for a project (small tweaks, edge cases, etc can still be flushed out after meeting), and review user flows and questions with wider product team. It is expected that intermittent discussions and reviews with the product and technical teams take place prior to this meeting via Figma comments or Slack threads. However, there is still a time when development needs to start, and a formal review meeting with the relevant parties is called. To make the most of this meeting, designer should resolve as many questions as possible via Figma comments, slack discussions, and individual research ahead of time, and reserve synchronous time to meatier discussions.
+
+**Output**: Scoping decisions on work with a rough outline of Milestones. It's possible, if the team feels ready, that the first Milestone meeting takes place during this session, pending attendee discussion and decisions.
+
+>>>>>>> Stashed changes
```
Comment (jdjkelly) [inline docs/process/development.md:237]: Accidental merge conflict:

```suggestion
#### Design review & scoping
* **Duration**: As long as it takes (typically starting with 90 minutes)

**Purpose**: To denmark the end of large scope design work for a project (small tweaks, edge cases, etc can still be flushed out after meeting), and review user flows and questions with wider product team. It is expected that intermittent discussions and reviews with the product and technical teams take place prior to this meeting via Figma comments or Slack threads. However, there is still a time when development needs to start, and a formal review meeting with the relevant parties is called. To make the most of this meeting, designer should resolve as many questions as possible via Figma comments, slack discussions, and individual research ahead of time, and reserve synchronous time to meatier discussions.

**Output**: Scoping decisions on work with a rough outline of Milestones. It's possible, if the team feels ready, that the first Milestone meeting takes place during this session, pending attendee discussion and decisions.

```

**Review by jdjkelly — CHANGES_REQUESTED** [review]

File: docs/process/development.md [hunk 2]
```diff
@@ -225,6 +225,13 @@ Details on pairing here.
 **Output**:  Ongoing notes. Action items on specific regulatory changes as needed.
 
 ### Ad-hoc
+#### Design review & scoping
+* **Duration**: As long as it takes (typically starting with 90 minutes)
+
+**Purpose**: To denmark the end of large scope design work for a project (small tweaks, edge cases, etc can still be flushed out after meeting), and review user flows and questions with wider product team. It is expected that intermittent discussions and reviews with the product and technical teams take place prior to this meeting via Figma comments or Slack threads. However, there is still a time when development needs to start, and a formal review meeting with the relevant parties is called. To make the most of this meeting, designer should resolve as many questions as possible via Figma comments, slack discussions, and individual research ahead of time, and reserve synchronous time to meatier discussions.
```
Comment (angelaliu22) [inline docs/process/development.md:231]:
```suggestion
**Purpose**: To demark the end of large scope design work for a project (small tweaks, edge cases, etc can still be flushed out after meeting), and review user flows and questions with wider product team. It is expected that intermittent discussions and reviews with the product and technical teams take place prior to this meeting via Figma comments or Slack threads. However, there is still a time when development needs to start, and a formal review meeting with the relevant parties is called. To make the most of this meeting, designer should resolve as many questions as possible via Figma comments, slack discussions, and individual research ahead of time, and reserve synchronous time to meatier discussions.
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
 # <img src="./docs/assets/logo-alt.png" height="60px" align="center" alt="Flexpa logo"> Flexpa Operating System
 
+## Contributing to the OS
```
Comment (amjarruda) [inline README.md:3]: Maybe a step 0 where you seek feedback where appropriate from other Flexpals before creating a PR.

**Review by amjarruda — APPROVED** [review]
Adding a step zero that is clear could be helpful considering team discussion via GitHub isn't as free-flowing and real-time as Slack allows for.

File: README.md (hunk 3 above)
Comment (angelaliu22) [reply README.md:3]: good call! I just added one - what do you think of this wording?

**Pull request merged**